### ISSUE TITLE
Text windows often have a clipped top line making them hard to read.

### ISSUE THREAD

**Issue by Mostelin** [christofmuc/KnobKraft-orm#213]
I've noticed this before and not mentioned it.

In certain windows, the first line of text appears cut off, so that the f character, for example, looks slightly more like a plus sign. This occurs in:

Log window
MIDI log window
Compare two patches pop-up

**Comment by christofmuc** [comment]
Yes, that is a long standing but I think in JUCE, the library I use. I haven't really assembled enough anger about it to start trying to fix it, as it should be working out of the box. Let's keep this here as a reminder, maybe I can encourage them to look at it.

**Comment by christofmuc** [comment]
I think this is not new, I have seen this problem before also in the 1.x versions.
